src/model_config_tests/qa/test_access_esm1p5_config.py

```diff
@@ -0,0 +1,167 @@
+
+        assert field_set == expected, error_field_incorrect(
+            field, "metadata.yaml", "sequence", expected
+        )
+
+    @pytest.mark.parametrize(
+        "field,expected",
+        [
+            ("nominal_resolution", VALID_NOMINAL_RESOLUTION),
+            ("reference", VALID_REFERENCE),
+        ],
+    )
+    def test_metadata_field_equal_expected_value(self, field, expected, metadata):
+        assert field in metadata and metadata[field] == expected, error_field_incorrect(
+            field, "metadata.yaml", expected
+        )
+
+    def test_config_start(self, branch, config):
+        assert (
+            "calendar" in config
+            and config["calendar"] is not None
+            and "start" in config["calendar"]
+            and config["calendar"]["start"] is not None
+        ), error_field_nonexistence("calendar.start", "config.yaml")
+
+        start: dict[str, int] = config["calendar"]["start"]
+
+        if branch.config_scenario == "preindustrial":
+            assert start == VALID_PREINDUSTRIAL_START, error_field_incorrect(
+                "calendar.start", "config.yaml", VALID_PREINDUSTRIAL_START
+            )
+        elif branch.config_scenario == "historical":
+            assert start == VALID_HISTORICAL_START, error_field_incorrect(
+                "calendar.start", "config.yaml", VALID_HISTORICAL_START
+            )
+        else:
+            pytest.fail(f"Cannot test unknown scenario {branch.config_scenario}.")
+
+    def test_config_runtime(self, config):
+        assert (
+            "calendar" in config
+            and config["calendar"] is not None
+            and "runtime" in config["calendar"]
+            and config["calendar"]["runtime"] is not None
+        ), error_field_nonexistence("calendar.runtime", "config.yaml")
+
+        runtime: dict[str, int] = config["calendar"]["runtime"]
+
+        assert runtime == VALID_RUNTIME, error_field_incorrect(
+            "calendar.runtime", "config.yaml", VALID_RUNTIME
+        )
+
+    def test_config_restart_freq(self, config):
+        assert (
+            "restart_freq" in config and config["restart_freq"] is not None
+        ), error_field_nonexistence("restart_freq", "config.yaml")
+        assert config["restart_freq"] == VALID_RESTART_FREQ, error_field_incorrect(
+            "restart_freq", "config.yaml", VALID_RESTART_FREQ
+        )
+
+    def test_mppncombine_fast_collate_exe(self, config):
+        # TODO: We don't check for high resolution here like we do in the ACCESS-OM2 version of the test. Should we?
+        pattern = r"/g/data/vk83/apps/mppnccombine-fast/.*/bin/mppnccombine-fast"
+        if "collate" in config:
+            assert re.match(
+                pattern, config["collate"]["exe"]
+            ), "Expect collate executable set to mppnccombine-fast"
+
+            assert config["collate"][
+                "mpi"
+            ], "Expect `mpi: true` when using mppnccombine-fast"
```

Comment:
Oh bugger. Just realised we shouldn't be testing `mppnccombine-fast`. We want to use the `mppnccombine` build by `spack` which looks like this

https://github.com/ACCESS-NRI/model-config-tests/blob/7053608e57c64864fe9f1b135e472eb384435e04/tests/resources/access-om2/configurations/release-1deg_jra55_ryf/config.yaml#L77C8-L77C26

Reply:
Updated in https://github.com/ACCESS-NRI/model-config-tests/pull/41/commits/c4d8a7d130b1a3764c34ba3c7cd0cbd7cb7971f8